products/deno.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Deno
+category: framework
+tags: javascript-runtime
+permalink: /deno
+versionCommand: deno --version
+releasePolicyLink: https://github.com/denoland/deno/releases
+changelogTemplate: https://github.com/denoland/deno/releases/tag/v__LATEST__
+releaseDateColumn: true
+
+identifiers:
+-   purl: pkg:docker/denoland/deno
+-   purl: pkg:github/denoland/deno
+-   repology: deno
+
+auto:
+  methods:
+  -   git: https://github.com/denoland/deno.git
+
+releases:
+-   releaseCycle: "2"
+    releaseDate: 2024-10-09
+    eol: false
+    latest: "2.1.4"
+    latestReleaseDate: 2024-12-11
+
+---
+
+> [Deno](https://deno.com) is a JavaScript, TypeScript, and WebAssembly runtime with
+> secure defaults and a great developer experience. It's built on V8, Rust, and Tokio.
+
+Deno support policy can be found on the
+[Stability and releases documentation page](https://docs.deno.com/runtime/fundamentals/stability_and_releases/).
```

Comment:
There needs to be a clear summary of the upstream policy here, instead of just a link. Things to include:

- Release Cadence
- Support policy
- Stability Guarantee
- Release Channels, including the exact guarantees made by LTS releases.
- Versioning Policy(?). Does deno use semver?
- Standard Library stability.

https://endoflife.date/nodejs might be a good reference.

Reply:
- [x]     Release Cadence
- [x]     Support policy
- [x]     Stability Guarantee
- [x]     Release Channels, including the exact guarantees made by LTS releases.
- [x]     Versioning Policy(?). Does deno use semver?
- [x]     Standard Library stability.

eaaa5d0